ssd_model:
  graph_file:
    name: MobileNetSSDV1_NCSDKV1_4Category_97000.graph
  threshold:
    value: 0.25
  detection_classes:
    names:
      - background
      - person
      - car
      - bicycle

